frontend/src/components/ExampleTestButton/Button.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 
+/**
+ *
+ * @param {*} text
+ * @return {*}
+ */
 export default function Button({text}) {
-    return (
-        <div>
-            <button>{text}</button>
-        </div>
-    );
-}
```

Comment:
I'm guessing these changes are a result of running the linter?

Reply:
Yes